ui/src/views/ForgotPassword.vue

```diff
@@ -140,10 +140,6 @@ export default defineComponent({
 </script>
 
 <style>
-.full-height {
-  height: 100vh;
-}
-
 .v-field__append-inner {
   cursor: pointer;
 }
```

Comment:
Should it be removed as well?

Reply:
Yes, it should be removed. It seems to have been included to fix a specific issue (vuetify alpha?). However, it no longer serves any practical purpose